projects/packages/sync/src/class-json-deflate-array-codec.php

```diff
@@ -41,7 +41,7 @@ public function encode( $object ) {
 	 * @return array|mixed|object
 	 */
 	public function decode( $input ) {
-		return $this->json_unserialize( gzinflate( base64_decode( $input ) ) ); // phpcs:ignore WordPress.PHP.DiscouragedPHPFunctions.obfuscation_base64_decode
+		return $this->json_unserialize( @gzinflate( base64_decode( $input ) ) ); // phpcs:ignore
```

Comment:
Let's not use generic `phpcs:ignore`. Better to be clear about what exactly we're ignoring.

Also, it seems like it might make more sense to store the result of `gzinflate` in a variable and check for `false` so we could skip trying to `json_unserialize` that.

Reply:
Thank you! I agree. Addressed these in 436ac0e747.